src/main/java/in/zapr/druid/druidry/filter/DruidFilter.java

```diff
@@ -18,9 +18,11 @@
 
 package in.zapr.druid.druidry.filter;
 
+import lombok.EqualsAndHashCode;
```

Comment:
If this works then can we use it in BoundFilter too?

Reply:
Will update it a separate patch. Need to fix this at lot of places.